### ISSUE TITLE
Hide info messages "deleting peer with id" from dmesg

### ISSUE THREAD

**Issue by oukb** [OpenVPN/ovpn-dco#53]
Hi

How can I hide dmesg/syslog messages such as `tun*: deleting peer with id 0, reason 2` or move it to openvpn.log?

**Comment by ordex** [comment]
It is printed on the INFO kernel loglevel, so you should set your whole kernel log level to something lower than that, but it is not recommended. Does it bother you? I thought it is useful to check when and why dco is deleting a peer.

OpenVPN in userspace should already separately log this kind of activity (when the right log level is enabled), but the fact that userspace is sending a command to DCO doesn't mean that DCO will properly execute it. This is why having that message in the kernel log is still useful.

**Comment by oukb** [comment]
Exactly
But maybe add some option to enable/disable these messages?
when there is nothing and suddenly these messages start constantly appearing

**Comment by ordex** [comment]
Yeah, ideally the debugging system will be extended/improved in the future. If you have any specific idea, feel free to drop a patch!
